index.md

```diff
@@ -65,19 +65,13 @@ author: "Andrei Guevorkian"
 - Assigning values and understanding variable scope.
 - Activity: Designing programs that make use of various data types and variables.
 
-### Chapter 6: Modularizing Your Code with Functions
+### [Chapter 6: Organizing Information with Data Structures](./programming-fundamentals/chapter6)
```

Comment:
The actual `chapter6.md` file shows another title in its front-matter. Which one is correct?

Reply:
You're right, sorry about that.